README.md

```diff
@@ -30,9 +30,10 @@ Or with BibTeX as:
 ```
 
 ## Contact
-Jos Cooper - jos.cooper@stfc.ac.uk \
+Jos Cooper - jos.cooper@ess.eu \
+Sjoerd Stendahl - sjoerd.stendahl@physics.uu.se \
 James Durant - james.durant@warwick.ac.uk \
-Lucas Wilkins - lucas@lucaswilkins.com
+Lucas Wilkins - lucas.wilkins@stfc.ac.uk
```

Comment:
Bit of a mystery why this is on the change-list, as this is already in main. The other of those cases disappeared when I rebased to a different branch, and then rebased back...